fdbserver/DataDistribution.actor.cpp

```diff
@@ -1980,6 +2016,7 @@ void runAuditStorage(Reference<DataDistributor> self,
 	    .detail("AuditType", audit->coreState.getType())
 	    .detail("Context", context);
 	addAuditToAuditMap(self, audit);
+	self->addActor.send(audit->errorOut.getFuture());
```

Comment:
What is this for?

Reply:
AuditStorageCore is running background. When moveKey_conflict happens in AuditStorageCore, we want this error is sent to DD and make DD restart.